tests/config/junos/BGP_NEIGHBOR_STATE_CHANGED/active_to_connect/syslog.msg

```diff
@@ -0,0 +1 @@
+<149>Jun 21 14:03:12  vmx01 rpd[2902]: RPD_BGP_NEIGHBOR_STATE_CHANGED: BGP peer 1.1.1.1 (External AS 2222) changed state from Active to Connect (event AnEvent) (instance master)
```

Comment:
It would be good if you could check the actual value the device sends for this PRI: this will be reflected in the expected output from `yang.json`. Now, the documentation is auto-generated from these mock files, as examples, e.g., http://napalm-logs.readthedocs.io/en/latest/messages/NTP_SERVER_UNREACHABLE.html, so it would be good to have the `facility` and `severity` in the examples as close as possible to the real values, so the users know what to expect.